src/Core/System/CustomField/Aggregate/CustomFieldSet/CustomFieldSetDefinition.php

```diff
@@ -42,6 +43,7 @@ protected function defineFields(): FieldCollection
             new JsonField('config', 'config', [], []),
 
             new BoolField('active', 'active'),
+            (new IntField('position', 'position'))->addFlags(new Required()),
```

Comment:
flag required is an bc. Old code will not work until they add position field. Why not just adding default value?

Reply:
I've removed the flag and added a default value.